beam/tools/src/main/java/cz/o2/proxima/beam/tools/groovy/BeamStream.java

```diff
@@ -1118,7 +1118,7 @@ private static void registerCommonTypes(Kryo kryo, Repository repo) {
     for (Field f : obj.getClass().getDeclaredFields()) {
```

Comment:
this code seems weird... the most used serializer is protobuf, which should be registered as:

```
kryo.register(message, new ProtobufSerializer<>)
```

current code results into not using proto serialization when available

Reply:
unrelated to this PR.

Reply:
Most notably - protobuf is registered as Coder using `BeamStream#registerCodersForSchemes`. This just registers additional types in Kryo.